src/formatting/config.rs

```diff
@@ -19,21 +19,28 @@ pub struct DummyConfig {
 
 impl Config {
     pub fn with_default(self, default_full: &str) -> Result<Format> {
+        self.with_defaults(default_full, "")
```

Comment:
~~Setting short text to `""` by default is wrong: it will result in blocks being hidden when the bar decides without user's permission.~~

While I was writing that comment I realized two things:
- `short_text` can be set to `""` and it is equvavent to not setting it all all, thus (at the moment) `Format::short` doesn't need to be an `Option`.
- If the user is willing to hide the block to save some space, short text needs to be set to something weird, like `"<span/>"`. That's not ideal.

Reply:
I was thinking that user could set `short` to null/nil/undefined value to just "disable" it, but it's not possible with TOML.

> If the user is willing to hide the block to save some space, short text needs to be set to something weird, like "<span/>". That's not ideal.

I guess it would have to be something like `"$hidden"` value that's basically hard-coded to `"<span/>"` or another option for format
```toml
[block.format]
short_hidden = true
```